service/telemetry/telemetry_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package telemetry
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"go.opentelemetry.io/collector/config/configtelemetry"
+)
+
+func TestTelemetryConfiguration(t *testing.T) {
```

Comment:
Is this test related to the change?

Reply:
No, it was more in the side of testing the validation of the Telemetry Configuration